### ISSUE TITLE
Create app icon

### ISSUE THREAD

**Issue by souvikbasu** [shoonyatech/reactive-kit#15]
Create an icon based on the RxJS icon given at https://rxjs-dev.firebaseapp.com/guide/overview

**Comment by AvinashDalvi89** [comment]
We have to create icon for reactive-kit app based on RxJs icon https://rxjs-dev.firebaseapp.com/assets/images/logos/logo.png. You mean it should be combination of this icon and kit icon ?

**Comment by souvikbasu** [comment]
yes the base will be RxJS icon and some changes on top of that to indicate a debugging action or a tool kit
